### PR TITLE
creating artifact repos with terraform

### PR DESCRIPTION
**Pull request by kingman**
# Description

Please provide a description of the changes you have made in this PR.

# How has this been tested?

Please explain how you have tested the new changes.

# Checklist

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have successfully run the E2E tests, and have included the links to the pipeline runs below
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have updated any relevant documentation to reflect my changes
- [ ] I have assigned a reviewer and messaged them

# Pipeline run links:

### CODE REVIEW

File: infrastructure/terraform/modules/feature-store/main.tf [hunk 1]
```diff
@@ -20,4 +20,44 @@ locals {
   customer_lifetime_value_project_id = local.config_vars.bigquery.dataset.customer_lifetime_value.project_id
   source_root_dir                    = "../.."
   sql_dir                            = "${local.source_root_dir}/sql"
-}
\ No newline at end of file
+  builder_repository_id              = "marketing-data-engine-base-repo"
+  pipelines_images_repository_id     = "pipelines-images"
+}
+
+module "project_services" {
```
Comment (chmstimoteo) [inline infrastructure/terraform/modules/feature-store/main.tf:27]: enable other apis as per doc

**Review by chmstimoteo — COMMENTED** [review]
leaving chagnes

File: infrastructure/terraform/modules/feature-store/main.tf (hunk 1 above)
Comment (kingman) [reply infrastructure/terraform/modules/feature-store/main.tf:26]: done

**Review by chmstimoteo — APPROVED** [review]
LGTM!

**Pull request merged**